internal/dataplane/kongstate/kongupstreampolicy.go

```diff
@@ -173,6 +173,7 @@ func translateHealthchecks(healthchecks *configurationv1beta1.KongUpstreamHealth
 	return &kong.Healthcheck{
 		Active:  translateActiveHealthcheck(healthchecks.Active),
 		Passive: translatePassiveHealthcheck(healthchecks.Passive),
+		Thresholds: healthchecks.Thresholds,
```

Comment:
It should be `heatlthchecks.Threshold`. Also, since the types are different in CRD and Kong SDK (`*int64` and `*flaot64`), there should be a `translateThreshold` function to run the transferring like 

```
func translateThreshold(threshold *int64) *float64 {
    if threshold == nil {
        return nil
    }
    return lo.ToPtr(float64(*threshold))
}
```